### ISSUE TITLE
Add thickness parameter for transmittance calculation.

### ISSUE THREAD

**Issue by Norihito-Hiruma** [asam-ev/OpenMATERIAL#141]
To calculate the attenuation of transmitted light, the thickness parameter of the material is required. 
The attenuation of light is calculated using Lambert-Beer's law, which requires not only the complex refractive index but also the path length inside the material. 
The path length of light within the material can be calculated using the refractive index and the thickness of the material, hence the need for the thickness parameter.

![Attenuation calculations](https://github.com/user-attachments/assets/40911c76-a423-413f-996c-b1afbb0f0781)

**Comment by Norihito-Hiruma** [comment]
I understand that Rott-san's opinion regarding thickness information is as follows:
   If thickness information is held within the model, it can be utilized. 
   If it is not available, then the thickness information within the Geometry information in the asset file can be used.
Is my understanding correct?
As you pointed out, thickness information is indeed part of the Geometry data.
If my understanding above is correct, I think it's a good idea.
It is necessary to have a mechanism in the asset file to associate information related to the shape required for sensor simulation with 3D shape components (nodes, etc.).

**Comment by ClemensLinnhoff** [comment]
I also agree, that thickness is a useful information, but not a material parameter. Let's discuss, if/how we can incorporate thickness (and potentially other volumetric parameters) into the asset file @LudwigFriedmann.

**Comment by LudwigFriedmann** [comment]
3D assets in computer graphics are typically surface models and not volume models (as in CAD applications).
This means that only the surface is modeled and there's no information about material thickness provided.

I'd therefore rather see the thickness in the material definition.
Like that, materials of specific thickness can then be assigned to 3D assets.
Depending on the impact of thickness on other material properties, this could also be variable parameter in material definitions.

**Comment by ClemensLinnhoff** [comment]
I would propose to move this to a future version of OpenMATERIAL.

**Comment by Norihito-Hiruma** [comment]
Based on Ludwig-san's opinion, I have considered the this task again.
As  Ludwig-san's pointed out, our data is polygon data, which is a surface model.
Furthermore, considering that there are cases where thin films or other surface features are not represented as polygon shapes, I believe thickness information should be included in the material information.

**Comment by ekuehn-dspace** [comment]
We use thickness in our simulation and thus would like it to be part of the standard. It does not really matter to us where the parameter is stored as long as it can be accessed somehow.
I agree that strictly speaking thickness is not a material property per se but for practicality, it might be good to have it as a material parameter (at least for the time being).

**Comment by LudwigFriedmann** [comment]
We could introduce a thickness parameter in the material mapping file (as second parameter besides color code or name).
This concept has to be refined, though. I'm not sure if we can complete that within the two remaining weeks.

**Comment by ClemensLinnhoff** [comment]
I agree that this proposal definitely has merit. But I don't think we can get to a solution within 2 weeks alongside all the other open topics.

**Comment by Norihito-Hiruma** [comment]
Thank you for your opnion.
Since there is not much time to discuss in V1.0, I agree to discuss the thickness parameter in V2.0.

**Comment by LudwigFriedmann** [comment]
Closed by #471
